### PR TITLE
Feature: 🎸 Parsing GoFundMe & Upload to Airtable

### PR DESCRIPTION
**Pull request by jaysonjphillips**
This brach includes the csv module for the RBB uploader. 
We create a new converter in the `/converters` folder, with an object that maps the csv fields to the Airtable fields. 

Also added a couple more options on the command line for passing in the file and the source (so we know which converter to use). Currently supports GoFundMe csv and OfficialBlackWallStreet.

### CODE REVIEW

**Comment by jaysonjphillips** [comment]
Instructions to run the command are in the README

File: README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ npm install -g .
 upload-data -t Businesses
 ```
 
-This will allow you to use the `upload-data` command from your terminal.
\ No newline at end of file
+This will allow you to use the `upload-data` command from your terminal.
```
Comment (llexical) [inline README.md:16]: The readme doesn't seem to have much added to it to explain what has changed/what commands are now avaliable.

Comment (llexical) [reply README.md:16]: Could the readme link to the docs file?

File: src/airtable-functions/create.js [hunk 2]
```diff
@@ -4,29 +4,18 @@ const fancyLog = require('../utils').fancyLog
...
-    },
-  ]
+  const listToUpload = await parseData(csvFile, csvSource)
   listRecords(tablename, function createRecords(tableData) {
     const dedupedUploadList = []
     listToUpload.forEach(function(row) {
       const recordAlreadyExists = tableData.find(function(datum) {
         // TODO: (bmc) Need to dedupe against additional fields.
-        return row.fields.Name === datum.Name
+        return row.fields.Name === datum.Name || row.fields.Email === datum.Email
```
Comment (llexical) [inline src/airtable-functions/create.js:18]: I feel we should be likely deduping on website and email more than name, there are many we don't have a name for. Also Business Name is viable. Website is pretty much a guaranteed unique though if we have it. Email and Business Name though we are much more likely to have.

Comment (brentmclark) [reply src/airtable-functions/create.js:18]: Fully agree.  I wanted to get this change set up last night, so I stopped at the point of proper deduplication.  It's on me to wrap that up in a separate PR as long as that's cool with you?

File: README.md (hunk 1 above)
Comment (brentmclark) [reply README.md:16]: Yep, I'll definitely tidy up the docs in a follow-up PR.  This PR represents @jaysonjphillips and I getting the repo stood up, so things are a little rocky right now.

**Comment by brentmclark** [comment]
@llexical your two comments will be addressed in separate PRs yet tonight.  I am going to get this one merged so @jaysonjphillips can move forward.

**Pull request merged**